core/bin/external_node/src/main.rs

```diff
@@ -92,6 +94,7 @@ impl Component {
                 Component::WsApi,
                 Component::Tree,
                 Component::Core,
+                Component::ContractVerificationApi,
```

Comment:
I don't think contract verifier should be in `all`, normally it's not required

Reply:
Updated.